Create venly-widget.md (#387)
0xPolygon/polygon-docs: docs/tools/wallets/venly/venly-widget.md

# Venly Widget

> Venly Widget is a powerful javascript SDK created to streamline everyday blockchain tasks. Its purpose is to enable functionalities otherwise restricted due to security implications, such as creating signatures. By encapsulating Venly's extensive capabilities within a user-friendly JavaScript layer, Venly Widget empowers developers and simplifies the development process.

Reviewer: ```suggestion
Venly Widget is a JavaScript SDK created to streamline everyday blockchain tasks. Its purpose is to enable functionalities otherwise restricted due to security implications, such as creating signatures. By encapsulating Venly's extensive capabilities within a user-friendly JavaScript layer, Venly Widget empowers developers and simplifies the development process.
```

> > If you are new to Web3 and don't have experience with blockchain technologies, we recommend you use the Venly Widget natively for a better developer experience.

Reviewer: You might need to do an admonition here .. please spin it up and check ... 

I recommend:

!!! tip

> ## Create Polygon wallets with Venly widget

Reviewer: ```suggestion
## Create Polygon wallets with Venly Widget
```

Venly Wallet allows you to create and manage wallets on the Polygon network. You can send and receive MATIC, Polygon NFTs, and ERC20 tokens. Apart from this, the Venly Wallet also supports:

- Multiple blockchains.
- Native token swap functionality.
- Signing messages (including EIP712 message).
- Calling smart contracts.
- Importing wallets.

## Look and feel

As the Widget is a product that incorporates a user interface (UI), let's look at how some of the more regular flows would appear for an end user.

### NFT transfer

The application prompts users to transfer an NFT from their wallet to a different destination in this flow.

> ![image](https://github.com/0xPolygon/polygon-docs/assets/139292301/da696d64-9dbc-4a1c-9527-ae91bfd19cb0)

Reviewer: Can you give the images better names for SEO please.

### Token transfer

The application prompts the user to transfer a token from their wallet to a different destination in this flow.

![image](https://github.com/0xPolygon/polygon-docs/assets/139292301/ae45b544-8ee3-4cfc-94f6-0cc430658f98)

> ## Different integration options

Reviewer: ```suggestion
## Integration options
```

Multiple integration options are available to incorporate the Venly Widget into your application. Here is a brief overview of some of these options:

> 1. [Native Integration with Venly SDK](https://docs.venly.io/docs/widget-overview): This approach involves utilizing the Venly SDK directly within your application's codebase. It allows you to access the full functionality of the Venly Widget and customize its behavior according to your requirements.
> 2. [Ethers.js Integration](https://docs.venly.io/docs/ethersjs): You can integrate the Venly Widget with your application using the popular Ethers.js library. This involves utilizing the Ethers.js API to interact with the Venly Widget and manage user wallets, transactions, and other blockchain-related operations.
> 3. [Web3Modal Integration](https://docs.venly.io/docs/web3modal-walletconnect) (WalletConnect): Web3Modal is a library that simplifies connecting to different wallet providers using standard protocols like WalletConnect. Integrating Web3Modal and WalletConnect enables users to interact with the Venly Widget and connect their wallets to your application seamlessly.

Reviewer: ```suggestion
1. [Native integration with Venly SDK](https://docs.venly.io/docs/widget-overview): This approach involves utilizing the Venly SDK directly within your application's codebase. It allows you to access the full functionality of the Venly Widget and customize its behavior according to your requirements.
2. [Ethers.js integration](https://docs.venly.io/docs/ethersjs): You can integrate the Venly Widget with your application using the popular Ethers.js library. This involves utilizing the Ethers.js API to interact with the Venly Widget and manage user wallets, transactions, and other blockchain-related operations.
3. [Web3Modal integration](https://docs.venly.io/docs/web3modal-walletconnect) (WalletConnect): Web3Modal is a library that simplifies connecting to different wallet providers using standard protocols like WalletConnect. Integrating Web3Modal and WalletConnect enables users to interact with the Venly Widget and connect their wallets to your application seamlessly.
```

These are just a few integration options to incorporate the Venly Widget into your application. The choice of integration method depends on your specific requirements, preferences, and the existing infrastructure of your application.

![image](https://github.com/0xPolygon/polygon-docs/assets/139292301/edcbb6b1-b17c-424f-bfa9-c8448cc8e441)

## When to choose what?

If you want to build your wallet app for users to interact with, you should use the [Wallet API](https://venly.readme.io/docs/overview).

> If you want to integrate an existing, full-fletched wallet solution, you can use the Venly Widget.  

Reviewer: ```suggestion
If you want to integrate an existing and complete wallet solution, you can use the Venly Widget.  
```

There are multiple ways to integrate it - natively or by using another library (which also uses the Venly Widget in the background):

![image](https://github.com/0xPolygon/polygon-docs/assets/139292301/7a6a6f02-10d9-48d3-83db-250d86406fff)

> 

Reviewer: Please spin up the site and check the table renders correctly. Also, the admonition from above ^^

> | Integration Type | Description                                                                                                                                                     | UI Flexibility                                                                                                                                                                                                | Blockchains          |

Reviewer: ```suggestion
| Integration type | Description                                                                                                                                                     | UI flexibility                                                                                                                                                                                                | Blockchains          |
```
